src/MonoTorrent/MonoTorrent.Client/Managers/TorrentManager.cs

```diff
@@ -406,7 +451,7 @@ void CreateRateLimiters ()
 
         #region Public Methods
 
-        internal void ChangePicker (PiecePicker picker)
+        public void ChangePicker(PiecePicker picker)
```

Comment:
This should not be public as it is not threadsafe for users of the library to invoke this directly. A threadsafe method does exist (`ChangePickerAsync`) so that's what people should be using.